TODO

```diff
@@ -0,0 +1,13 @@
+// snek TODO
+
+add optional game modifications
+add more ways to control the game
+add pause
+add easter egg
+add sound effects
+make the field explode on loose
+add lose screen
+
+create game menu
+
+add second player
```

Comment:
Ok, that could actually be **very nice!**

Reply:
yes